.github/workflows/merge-release-branch.yml

```diff
@@ -1,6 +1,11 @@
 name: Merge Release Into Master
 
-on: workflow_dispatch
+on: 
+  workflow_dispatch:
+    inputs:
+      version:
+        description: 'Release version'     
+        required: true
```

Comment:
Can you set the version from the Github UI?
If we would like I think we can extract the version from the package.json as well (could be an improvement for later)
see: https://github.com/firebase/firebase-admin-node/blob/a32195daa9848b261fe892d9f606152a40ff2915/.github/scripts/publish_preflight_check.sh#L61

Reply:
Thanks, I added this, if you want to take another look. I tested it out with the merge step commented out and it seems to work up to that point.